aws-opentelemetry-distro/src/amazon/opentelemetry/distro/aws_opentelemetry_configurator.py

```diff
@@ -281,25 +310,68 @@ def _customize_span_processors(provider: TracerProvider, resource: Resource) ->
     return
 
 
+def _customize_metric_exporters(metric_readers: List[MetricReader], views: List[View]) -> None:
+    if _is_application_signals_runtime_enabled():
+        system_metrics_scope_name = "opentelemetry.instrumentation.system_metrics"
+        if 0 == len(metric_readers):
+            _logger.info("Registered scope %s", system_metrics_scope_name)
+            views.append(View(meter_name=system_metrics_scope_name, aggregation=DefaultAggregation()))
+            views.append(View(instrument_name="*", aggregation=DropAggregation()))
+
+        otel_metric_exporter = ApplicationSignalsExporterProvider().create_exporter()
```

Comment:
Would it be more appropriate to call this `application_signals_metric_exporter`?

Reply:
Renamed.